HTAN.model.csv

```diff
@@ -84,7 +84,7 @@ Visium File Type,The file type generated for the visium experiment.,"reference p
 Run ID,A unique identifier for this individual run (typically associated with a single slide) of the spatial transcriptomic processing workflow.,,,,TRUE,Spatial Transcriptomics,,,
 Capture Area,"Area (or Capture Area) - One of the either four or two active regions where tissue can be placed on a Visium slide. Each area is intended to contain only one tissue sample. Slide areas are named consecutively from top to bottom: A1, B1, C1, D1 for Visium slides with 6.5 mm Capture Area and A, B for CytAssist slides with 11 mm Capture Area. Both CytAssist slides with 6.5 mm Capture Area and Gateway Slides contain only two slide areas, A1 and D1.","A, B, C, D, A1, B1, C1, D1",,,FALSE,Spatial Transcriptomics,,,
 Slide Version,Version of imaging slide used. Slide version is critical for the analysis of the sequencing data as different slides have different capture area layouts.,"V1, V2, V3, V4",,,FALSE,Spatial Transcriptomics,,,
-Slide ID,"For Visium, it is the unique identifier printed on the label of each Visium slide. The serial number starts with V followed by a number which can range between one through five and ends with a dash and a three digit number, such as 123. For CosMx, this refers to the loaded Flow Cell ID. For Xenium, this ID indicates the slide orientation, as it matches the relative location of the ID on the physical Xenium slide.",,,,FALSE,Spatial Transcriptomics,,,

```

Comment:
Looks like we are reverting to older changes here. Make sure branch up to date with main